sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_version.py

```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
-VERSION = "4.2.2"
+VERSION = "4.2.2b1"
```

Comment:
These packages have new API, they should be betas of the next major version:
```suggestion
VERSION = "4.3.0b1"
```